pad_api_data/pad_etl/data/dungeon.py

```diff
@@ -30,88 +30,80 @@ def __init__(self, raw: List[Any]):
+            modifiers = Modifier()
+
+        self.possible_drops = modifiers.possibleDrops
+        self.entry_requirement = modifiers.entryRequirement
+        self.required_dungeon = modifiers.requiredDungeon
+        self.remaining_modifiers = modifiers.remainingModifiers
+        self.stat_modifiers = modifiers.modifiers
+        self.enhanced_type = modifiers.enhancedType
+        self.enhanced_attribute = modifiers.enhancedAttribute
+        self.messages = modifiers.messages
+        self.fixed_team = modifiers.fixedTeam
+        self.remaining_modifiers = modifiers.remainingModifiers
+        self.test_score = modifiers.score
+
+        pos = modifiers.pos
+
+        # for debugging just in case otherwise we can negate
+        self.original_fields = raw[pos:]
 
         self.flags = int(raw[pos])
-        self.remaining_fields = raw[pos+1:]
-
-        # Modifiers parsing doesn't seem to always work
-        # Hacked up version for dungeon modifiers, needed for
-        # enemy parsing.
-        self.modifiers_clean = {
-            'hp': 1.0,
-            'atk': 1.0,
-            'def': 1.0,
-        }
-
-        for field in self.remaining_fields:
-            if 'hp:' in field or 'at:' in field or 'df:' in field:
-                for mod in field.split('|'):
-                    if mod.startswith('hp:'):
-                        self.modifiers_clean['hp'] = float(mod[3:]) / 10000
-                    elif mod.startswith('at:'):
-                        self.modifiers_clean['atk'] = float(mod[3:]) / 10000
-                    elif mod.startswith('df:'):
-                        self.modifiers_clean['def'] = float(mod[3:]) / 10000
-                break
-
-
-        # Modifiers parsing also seems to skip fixed teams sometimes.
-        # Hacked up version for just that here.
-        self.fixed_team = {}
-
-        for field in self.remaining_fields:
-            if not 'fc1' in field:
-                continue
-            for sub_field in field.split('|'):
-                if not sub_field.startswith('fc'):
-                    continue
-                idx = int(sub_field[2])
-                contents = sub_field[4:]
-                details = contents.split(';')
-                full_record = len(details) > 1
-                self.fixed_team[idx] = {
-                    'monster_id': details[0],
-                    'hp_plus': details[1] if full_record else 0,
-                    'atk_plus': details[2] if full_record else 0,
-                    'rcv_plus': details[3] if full_record else 0,
-                    'awakening_count': details[4] if full_record else 0,
-                    'skill_level': details[5] if full_record else 0,
-                }
+        self.remaining_fields = raw[pos + 1:]
+        #
+        # # Modifiers parsing doesn't seem to always work
+        # # Hacked up version for dungeon modifiers, needed for
+        # # enemy parsing.
+        # self.modifiers_clean = {
```

Comment:
This field is currently used elsewhere and it doesn't look like you've replaced it or changed the usage. 

Committing this will break things, which is not good.